src/app/pages/Market/crypto-exchange/steps/ConvertationTracker/components/CompletedStatusInfoBlocks.tsx

```diff
@@ -45,10 +46,14 @@ export const CompletedStatusInfoBlocks = memo(() => {
         <InfoRaw bottomSeparator title="youSent">
           <div className="flex flex-row gap-x-0.5">
             <span className="p-1 text-font-description-bold">
-              <Money smallFractionFont={false} tooltipPlacement="bottom">
+              <Money
+                cryptoDecimals={exchangeData.amount.length > 12 ? 2 : 6}
+                smallFractionFont={false}
+                tooltipPlacement="bottom"
+              >
                 {exchangeData.amount}
               </Money>{' '}
-              {exchangeData.coinFrom.coinCode}
+              {getCurrencyDisplayCode(exchangeData.coinFrom.coinCode)}
```

Comment:
Code like this seems to appear often in the files that are related to Exolix. Please consider creating a separate component and using it wherever it is relevant